### ISSUE TITLE
Re-enable Swift 2.0 optimizations in Release

### ISSUE THREAD

**Issue by NachoSoto** [ReactiveCocoa/ReactiveCocoa#2331]
We disabled them in an earlier Xcode 7 beta, but we should check if it compiles now (also using `-Owholemodule`)

**Comment by NachoSoto** [comment]
I'm happy to do this, I just thought of it right now and wanted to file the issue before I forgot!

**Comment by NachoSoto** [comment]
I just did this in the `Swift 2` branch, and tests pass :tada:.

I'm waiting on https://github.com/jspahrsummers/xcconfigs/pull/43 and https://github.com/jspahrsummers/xcconfigs/pull/42 to be landed so that I can just update the `xcconfig` pointer and remove our setting override.

**Comment by neilpa** [comment]
Awesome, will be nice to have these back on.

**Comment by NachoSoto** [comment]
Cool, @jspahrsummers merged the `xcconfig` changes, but I'd prefer if we can land #2333 first to avoid conflicts with the changed `xcconfig` pointer.

**Comment by neilpa** [comment]
Those changes look good. Just merge them when you're ready.

**Comment by NachoSoto** [comment]
Done!
